swcc/swcc/models/project.py

```diff
@@ -366,9 +366,13 @@ def download(self, folder: Union[Path, str]):
         r: requests.Response = session.get(f'{self._endpoint}/{self.id}/download/')
         raise_for_status(r)
         data = r.json()
-        for path, url in data['download_paths'].items():
+        files = data['download_paths']
+        print(f'Downloading {len(files)} files...')
+        print_progress_bar(0, len(files))
+        for index, (path, url) in enumerate(files.items()):
             file_item: FileType = FileType(url=url)
             file_item.download(Path(folder, *path.split('/')[:-1]))
+            print_progress_bar(index + 1, len(files))
```

Comment:
Before, we had a solitary `print()` after everything to go to a new line.  Can you add that back, too.

Reply:
Sure, added in [6ed0f2e](https://github.com/girder/shapeworks-cloud/pull/301/commits/6ed0f2e6626a89fdb1053c1081bcc4cfaceb774f)